lib/src/app.dart

```diff
@@ -204,6 +204,9 @@ class _AppState extends ConsumerState<Application> {
           localizationsDelegates: AppLocalizations.localizationsDelegates,
           supportedLocales: kSupportedLocales,
           onGenerateTitle: (BuildContext context) => 'lichess.org',
+          locale: ref.watch(
```

Comment:
We have too many `generalPreferencesProvider.select` now. I think it's better to have one single
```dart
final generalPrefs = ref.watch(generalPreferencesProvider);
```

at the beginning of `build`. (It is better to put `ref.watch` at the beginning so we know what provider are listened to).